util/gnmi.go

```diff
@@ -134,6 +134,41 @@ func FindPathElemPrefix(paths []*gpb.Path) *gpb.Path {
 	}
 }
 
+// pathElemMatchesQuery returns true if the given concrete (no wildcard)
```

Comment:
Can you clarify `concrete` here? I think that you're trying to say if the given query element matches the reference element -- but the `no wildcard` disclaimer is throwing me off, because it seems like wildcards are handled?